multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -107,35 +802,46 @@ fn verify_bill_payment() {
             make_init_config(cluster.chain),
         )
         .unwrap();
-    let consuming_payable_dao = PayableDaoReal::new(consuming_node_connection);
+    let consuming_node_payable_dao = PayableDaoReal::new(consuming_node_connection);
     open_all_file_permissions(consuming_node_path.clone().into());
     assert_eq!(
-        format!("{}", &contract_owner_wallet),
-        "0x5a4d5df91d0124dec73dbd112f82d6077ccab47d"
+        format!("{}", &consuming_node_wallet),
+        "0x7a3cf474962646b18666b5a5be597bb0af013d81"
     );
     assert_eq!(
         format!("{}", &serving_node_1_wallet),
-        "0x7a3cf474962646b18666b5a5be597bb0af013d81"
+        "0x0bd8bc4b8aba5d8abf13ea78a6668ad0e9985ad6"
     );
     assert_eq!(
         format!("{}", &serving_node_2_wallet),
-        "0x0bd8bc4b8aba5d8abf13ea78a6668ad0e9985ad6"
+        "0xb329c8b029a2d3d217e71bc4d188e8e1a4a8b924"
     );
     assert_eq!(
         format!("{}", &serving_node_3_wallet),
-        "0xb329c8b029a2d3d217e71bc4d188e8e1a4a8b924"
+        "0xb45a33ef3e3097f34c826369b74141ed268cdb5a"
     );
     let now = SystemTime::now();
-    consuming_payable_dao
-        .more_money_payable(now, &serving_node_1_wallet, amount)
+    consuming_node_payable_dao
```

Comment:
You can migrate these 3 setting up statements into a function which takes `dao` and `global_config` to create debts inside the Consuming Node's DB.